content/docs/guides/crosswalk/kubernetes/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Pulumi Crosswalk for Kubernetes"
+linktitle: Crosswalk for Kubernetes
+menu:
+  userguides:
+    parent: guides
+    identifier: crosswalk-kubernetes
+
+aliases: ["/docs/reference/crosswalk/kubernetes/"]
+---
+
+Pulumi Crosswalk for Kubernetes is a collection of industry standard
+best-practices and tasks for managing Kubernetes and its infrastructure in production.
+
+In this guide we will show you, using batteries-included examples, how to provision
+and use these technologies together in a way that maintains high release velocity, without
+sacrificing security, governance, or stability.
+
+This guide is for provisioning and configuring production-grade Kubernetes
+clusters. If you are just getting started with Pulumi and Kubernetes, the [Get
+Started guide][k8s-get-started] is a better place to start.
+
+<center><img src="/images/docs/quickstart/kubernetes/cake.svg" width="670"></center>
+
+## Getting Started
```

Comment:
A few lines up, it's mentioned that the Get Started guide is a good place to start, so you can probably remove this heading and the Get Started bullet below it, unless you wanted to model the section more on the way it's handled on the Crosswalk for AWS index page (which might be a good way to go, just to keep these two structurally similar).

The Introduction page feels like it belongs somewhere, though. Perhaps work its content into this page somehow?

Reply:
Done. I've removed the dupe getting started links, and consolidated the Intro/ Production Architecture for Teams into the crosswalk for k8s index page.